src/casync-http.c

```diff
@@ -32,6 +33,39 @@ typedef enum ProcessUntil {
         PROCESS_UNTIL_FINISHED,
 } ProcessUntil;
 
+static CURLcode robust_curl_easy_perform(CURL *curl) {
+        useconds_t sleep_time = 100000;
+        uint trial = 1;
+        uint limit = 10;
```

Comment:
please, just use "unsigned" here. uint is a weird, redundant type...

Reply:
ack